src/repository/photos.py

```
@@ -0,0 +1,67 @@
+from typing import List, Type
+from sqlalchemy import and_
+from sqlalchemy.orm import Session
+from src.database.models import Photo, User, Tag
+from src.schemas import PhotoModel, PhotoResponse, TagModel, TagsPhoto
+from datetime import date, timedelta
+
+from fastapi import UploadFile
+
+
+async def create_photo(text: str, url: str, db: Session) -> Photo:
+
+    photo = Photo(
+        user_id=1,
+        url=url,
+        description=text
+    )
+    db.add(photo)
+    db.commit()
+    db.refresh(photo)
+    return photo
+
+
+async def read_photo(photo_id: int, db: Session) -> Photo | None:
+    return db.query(Photo).filter(Photo.id == photo_id).first()
+
+
+async def update_photo(photo_id: int, url: UploadFile, description: str, db: Session) -> Photo | None:
+
+    photo = db.query(Photo).filter(Photo.id == photo_id).first()
+    if photo:
+        photo.url = url
+        photo.description = description
+        db.commit()
+    return photo
+
+
+async def delete_photo(photo_id: int, db: Session) -> Photo | None:
```

Please return a status message or confirmation that the photo has been deleted.Additionally,please to raise an exception if the photo is not found(this is what we discussed on the call in the evening)

---

the exception 'photo not found' is raised in routes/photos:delete_photo